### DISCUSSION TITLE
Linear Regression in Python

### DISCUSSION THREAD

**Discussion by fxpeterbs31** [teamalgoritma/community#439] [Q&A] [answered]
Halo, saya mau bertanya. Di R, kita bisa membuat model linear regression dengan menggunakan fungsi lm(). Kita juga bisa mendapatkan interpretasinya dengan menggunakan summary(). 

Apakah di python ada juga fungsi yang sama/serupa dengan fungsi lm() yang ada di R, serta apakah describe() disini bisa menggantikan fungsi summary() di python dalam konteks interpretasi model? Terima kasih.

**Comment by kevbow** [comment, marked as answer]
Dear @fxpeterbs31,

Pada python kita dapat menggunakan library statsmodels untuk membuat model linear yang memiliki interpretasi.
Berikut saya lampirkan dokumentasi resmi dari library ini dan beberapa penggunaannya:
- https://www.statsmodels.org/dev/examples/index.html

Demikian yang dapat saya sampaikan, semoga membantu.
